### ISSUE TITLE
A question about the paper

### ISSUE THREAD

**Issue by CurryxIaoHu** [RUCAIBox/Slow_Thinking_with_LLMs#32]
Thanks for your valuable work! But I have a question after reading this paper. What is the difference between "On-policy Training" and "On-policy Training"? And what is the correct setting in openrlhf to achieve "on-policy training"?

**Comment by Timothy023** [comment]
Hi,

On-policy training refers to using the whole sampled response to perform **once** gradient descent. In openrlhf, you can set the train_batch_size=rollout_batch_size*n_samples_per_prompt and set max_epochs=1 to achieve on-policy training
